### PR TITLE
SPOI-10711 Add documentation for security/authentication and system alerts

### PR DESCRIPTION
**Pull request by sanjaypujare**
@amberarrow @wvary pls review and merge as appropriate

### CODE REVIEW

**Comment by sanjaypujare** [comment]
@amberarrow @wvary  another reminder for PR

File: docs/dtmanage.md [hunk 1]
```diff
@@ -68,6 +68,56 @@ This can be useful when Hadoop configuration has changed or some other factor of
...
+
+Use this page to configure the authentication method. You can set the authentication method to be "None" or "Password" as shown in the following screenshot.
+
+![Security Configuration Page](images/dtmanage/security-screen1.png)
+
+Data Torrent RTS does support other authentication methods but they have to be configured through configuration files as described [here](dtgateway_security/#authentication)
+
+### System Alerts
+
+Use this page manage system alerts.
```
Comment (amberarrow) [inline docs/dtmanage.md]: page manage => page to manage

File: docs/dtmanage.md [hunk 2]
```diff
@@ -68,6 +68,56 @@ This can be useful when Hadoop configuration has changed or some other factor of
...
+
+![System Alerts page](images/dtmanage/system-alerts1.png)
+
+Click on the "+ create new alert" button to create an alert.
```
Comment (amberarrow) [inline docs/dtmanage.md]: using backquotes will highlight it better: `+ create new alert`

File: docs/dtmanage.md [hunk 3]
```diff
@@ -68,6 +68,56 @@ This can be useful when Hadoop configuration has changed or some other factor of
...
+
+An alert consists of
+
+ * a condition (a JavaScript expression)
+ * a list of recipient email addresses
+ * a threshold value in milliseconds
+ * a message, and
+ * an enabled/disabled flag
+
+The gateway periodically (every 5 seconds) processes all enabled alerts by evaluating the condition. If the condition evaluates to true, the alert is said to be "in effect".
```
Comment (amberarrow) [inline docs/dtmanage.md]: Is the 5 seconds configurable ? If so, can we say how to configure it ? If not, ok to leave as is.

File: docs/dtmanage.md [hunk 4]
```diff
@@ -68,6 +68,56 @@ This can be useful when Hadoop configuration has changed or some other factor of
...
+If the condition evaluates to false, the alert is said to be "out" (or "out of effect"). If the alert stays "in effect" for the duration specified as the threshold value,
+then the alert is triggered and the gateway sends an "in effect" email message to all the recipient email addresses.
+
+If a triggered alert goes "out of effect" then the gateway immediately sends an "out of effect" email message to all the recipient email addresses.
+
+The alert condition is specified as a JavaScript expression which are evaluated in the context of something called topics which are described [here](dtgateway_systemalerts/#alerts-and-topics).
```
Comment (amberarrow) [inline docs/dtmanage.md]: are evaluated => is evaluated

File: docs/dtmanage.md [hunk 5]
```diff
@@ -68,6 +68,56 @@ This can be useful when Hadoop configuration has changed or some other factor of
...
+As an example, for the "Application Memory Usage" template you need to provide the Application Name and Memory values as shown below:
+
+![System Alerts page](images/dtmanage/system-alerts4.png)
+
+You can click on the "Javascript Code" tab to see the generated JavaScript expression that corresponds to your alert template selection and provided values as shown below:
+
+![System Alerts page](images/dtmanage/system-alerts5.png)
+
+You can generate a test email to validate your alert by checking the "Send Test Email" check-box and clicking on the blue "Test" button. The test email is sent regardless of the true or false result
+of the JavaScript condition, if the evaluation has no errors.
```
Comment (amberarrow) [inline docs/dtmanage.md]: Is it worth pointing out that all this requires an SMTP server to be configured along with a link to that info ?

**Comment by amberarrow** [comment]
Made a couple of comments, rest looks good.

File: docs/dtmanage.md (hunk 3 above)
Comment (sanjaypujare) [inline docs/dtmanage.md]: 5 seconds is not configurable.

Comment (sanjaypujare) [inline docs/dtmanage.md]: Will use `` to highlight true and false but I am not sure if they should be considered as literal strings. Aren't they just boolean values in JavaScript just like in Java?

File: docs/dtmanage.md (hunk 5 above)
Comment (sanjaypujare) [inline docs/dtmanage.md]: Will do - but until the other PR is merged I don't have a page to link to !

**Comment by sanjaypujare** [comment]
@amberarrow comments incorporated for review and merge

File: docs/dtmanage.md (hunk 3 above)
Comment (amberarrow) [inline docs/dtmanage.md]: In non-technical prose, "true" and "false" are adjectives but here they are being used as nouns, so highlighting it in some way seems appropriate. For example, in https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html you'll see this sentence: "The Boolean object corresponding to the primitive value `true`." where the word "true" is in a different font.

If the sentence reads: "If the condition is true ..." then the usage is adjectival and no highlighting is necessary.

Comment (sanjaypujare) [inline docs/dtmanage.md]: Got it...

**Pull request merged**